### DISCUSSION TITLE
Pre heat timeout?

### DISCUSSION THREAD

**Discussion by darco415** [Jyers/Marlin#391] [Q&A] [answered]
When I choose warmup from the preheat menu, warms up fine, it runs indefinitely. When I choose, PLA, it heats up fine then shuts down after approx 5 minutes. Is this by design?
thxs

**Comment by Jyers** [comment, marked as answer]
From what I can tell nothing in the firmware would be doing this. It could be some safety feature I'm unaware of that is only triggered at certain temps, but I find that unlikely.

**Reply by darco415** [reply]
I pre heated w/o octoprint connected since I had a previous heater timeout plugin but it still times out after a few mins. So no one else has this issue?
I do self compile but I didnt mess with the preheat config...
thxs
